### PR TITLE
Create node

### LINKED ISSUE #8: Integrate node creation with draw model

### PR DESCRIPTION
**Pull request by sachingrover211**
Changes for showing a node with when create quantity node is clicked, and the dynamic position code is handled as well.

Resolves #8

### CODE REVIEW

File: www/js/main.js [hunk 1]
```diff
@@ -170,19 +170,28 @@ define([
 			menu.add("createQuantityNodeButton", function(e){
 				event.stop(e);
 				//give a fake id for now
```
Comment (jonwetzel) [inline www/js/main.js]: You can remove this comment now. :)  

Also remove the same comment from line 185, and the dead code from line 190.

File: www/js/model.js [hunk 2]
```diff
@@ -175,7 +218,31 @@ define([
 		};
 
 		obj.authored = lang.mixin({
+			addNode: function(options){
+				obj._updateNextXYPosition();
+				var newNode = lang.mixin({
+					ID: "id" + obj._ID++,
+					genus: "required",
+					accumulator: false,
```
Comment (jonwetzel) [inline www/js/model.js]: Why is this field called "accumulator"?  Shouldn't it be "dynamic" instead?

Comment (sachingrover211) [inline www/js/model.js]: Professor, in our discussion had said it to be accumulator. This is what is in the data file.

File: www/js/main.js (hunk 1 above)
Comment (sachingrover211) [inline www/js/main.js]: Removed

File: www/js/model.js (hunk 2 above)
Comment (jonwetzel) [inline www/js/model.js]: Ok!

**Pull request merged**